# ComputeRadius.srv
---
float32 radius